### PR TITLE
Support for 0.40.0 features

### PR DESCRIPTION
**Pull request by lucacasonato**
### do not merge until 0.40.0 lands and [deno-lambda](https://github.com/hayd/deno-lambda) is updated

### CODE REVIEW

**Comment by ry** [comment]
0.40.0 is released. Ping me when we can land this.

**Comment by lucacasonato** [comment]
Ready to land

File: api/docs.ts [hunk 1]
```diff
@@ -37,24 +37,22 @@ export async function handler(
...
   });
 
   let killed = false;
 
+  // Zeit timeout is 10 seconds for free tier: https://zeit.co/docs/v2/platform/limits
   const timer = setTimeout(() => {
     killed = true;
     proc.kill(Deno.Signal.SIGKILL);
-  }, 5000);
+  }, 9000);
```
Comment (ry) [inline api/docs.ts:55]: hmm...

File: package.json [hunk 2]
```diff
@@ -5,7 +5,8 @@
   "scripts": {
     "dev": "next dev",
     "build": "next build",
-    "start": "next start"
+    "start": "next start",
+    "format": "prettier **.tsx **.ts **.js **.json"
```
Comment (ry) [inline package.json:9]: can we use deno fmt ?

File: api/docs.ts (hunk 1 above)
Comment (lucacasonato) [reply api/docs.ts:55]: Deeply nested files take away while to download... Now's free limit is 10 seconds.

**Pull request merged**